src/module/actor/sheet/base.js

```diff
@@ -556,6 +560,80 @@ export class ActorSheetSFRPG extends ActorSheet {
 
     }
 
+    async _onOpenBrowser(event) {
+        event.preventDefault();
+        const filterType = event.currentTarget.dataset.type;
+        const classesToFilters = {
+            'mystic': 'myst',
+            'precog': 'precog',
+            'technomancer': 'tech',
+            'witchwarper': 'wysh'
+        };
+        let browser;
+        let activeFilters = {};
+        switch (filterType) {
```

Comment:
Changing the switch case to an else if seems to fix equipment, but not spells. Again, no idea why this broke unless you changed something

Reply:
Problem solved. Switch case was to be changed and `browser.refreshFilters` needed to be set to true.